src/api/game.js

```diff
@@ -0,0 +1,21 @@
+import { basicJsonHeader, handleResponse } from './tools';
+import { parseInput } from '../utils/game';
+
+const PERKINS_API_URL = process.env.REACT_APP_PERKINS_API_URL || 'http://localhost:5000';
+
+export const createGame = async (name, createdBy = {}) => {
+  const response = await fetch(`${PERKINS_API_URL}/api/games`, {
```

Comment:
Do you have an implementation of that API in a branch in the `server` repo? I'm tempted to suggest that we set the precedent that any UI PR dependent on an API should wait for the API feature to merge as well.

Reply:
Great point. It does have an accompanying server update to go with it, which I forgot to put a PR in for.